org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/serializer/SerializerFragment2.xtend

```diff
@@ -475,7 +474,6 @@ class SerializerFragment2 extends AbstractStubGeneratingFragment {
 				�IF partitions.size > 1�
 					�FOR partition : partitions.indexed�
 						private void init�partition.key�(�IGrammarAccess� access) {
```

Comment:
should we make the initx method empty then @szarnekow @lukasf98

Reply:
empty in the sense of `no args` ? Yes.

Reply:
yes. 

so

init1()
init2()
init3()